src/components/landingPage/index.tsx

```diff
@@ -1,16 +1,56 @@
 import React from 'react';
 
 export interface LandingPageProps {
-    
+
+}
+
+export interface LandingPageState {
+  info: String;
 }
 
- 
-class LandingPage extends React.Component<LandingPageProps> {
-    render() { 
-        return ( <div>
-            <h1>hello</h1>
-        </div> );
+class LandingPage extends React.Component<LandingPageProps, LandingPageState> {
+  Constructor(props: LandingPageProps){
+    this.state = {
+      info: "hello"
     }
+  }
+
+  // handleSignOut = async () => {
```

Comment:
There is a lot of commented code in this file. Is it still needed? Is this component even being used? If not, feel free to delete it.